datumaro/components/operations.py

```diff
@@ -86,8 +87,18 @@ def __call__(self, sources):
         raise NotImplementedError()
 
 class ExactMerge:
+    """
+    Merges few datasets using the "simple" algorithm:
+
+    - items are matched by (id, subset) pairs
+    - matching items share the data available (nothing + nothing = nothing,
```

Comment:
What is meant by "data" here? There's no such attribute in `DatasetItem`.

Reply:
Fixed